format/Flight.proto

```diff
@@ -360,3 +361,96 @@ message FlightData {
 message PutResult {
   bytes app_metadata = 1;
 }
+
+/*
+ * Request message for the "Close Session" action.
+ *
+ * The exiting session is referenced via a cookie header.
+ */
+message ActionCloseSessionRequest {
```

Comment:
Can we be consistent with the existing messages? https://github.com/apache/arrow/blob/main/format/Flight.proto#L191

Reply:
Sorry, somewhat confused here: Consistent as far as docs or as far as the message content?  The content here is implicit/OOB (cookie header) for the call vs FlightInfo being in-band...

Reply:
Ping

Reply:
Ah, sorry. I mean in terms of naming. Looks like I linked to the wrong place, I meant here as an example: https://github.com/apache/arrow/blob/3ad2d0ccd98192d32d54af371bad6b61ea3c4736/format/Flight.proto#L216

So just CloseSessionRequest here.